src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -358,6 +368,17 @@ class ErgoWalletActor(settings: ErgoSettings, boxSelector: BoxSelector)
         }
     }
 
+  /**
+    * A helper method which makes TrackedBox sequence out of boxes provided
+    */
+  private def boxesToFakeTracked(inputs: Seq[ErgoBox]): Iterator[TrackedBox] = {
```

Comment:
Why making separate method which is used only once and locally inside another function?

Reply:
To make generateTransactionWithOutputs shorter and simpler

Reply:
@kushti, I think it wouldn't be far less simpler if this piece of code was assigned to `val` inside `if` statement where it is used.

Reply:
@oskin1 as alternative I propose to move generating functions to a dedicated package (object)

Reply:
@oskin1 then helper functions even used once look pretty natural